tests/phpunit/RetryTest.php

```diff
@@ -167,7 +167,8 @@ private function getRetryConfig(): array
             'query' => 'SELECT * FROM sales',
             'outputTable' => 'in.c-main.sales',
             'incremental' => false,
-            'primaryKey' => null,
+            'primaryKey' => [],
+            'columns' => [],
```

Comment:
Toto pole sa pouziva priamo ... a teda tu musia byt hodnoty uz normalizovane, ... nejde to cez `ConfigDefinition`.

Reply:
Asi je tam predsa len chyba, ... idem to este chcecknut.

Reply:
Tak je to teda OK. 

Padalo to v `RetryTest::testRetryNetworkErrorFetchFailure` 

... a tam ide config priamo do `extractora`:
https://github.com/keboola/db-extractor-common/blob/dc64ed75125a429423cb822bed34f3acd20c3557/tests/phpunit/RetryTest.php#L526 ... a teda ten config musi byt uz normalizovany.


Na inych miesta toho testu ide config do `Applicaiton`, co ma zmiatlo, ... kedze tam by nemusel byt ten config normalizovany: https://github.com/keboola/db-extractor-common/blob/dc64ed75125a429423cb822bed34f3acd20c3557/tests/phpunit/RetryTest.php#L372


Teda tie povodne testy su trocha blbo napisane, ale je to OK.

Reply:
... na to "normalizovanie" configu a default values je aj test: https://github.com/keboola/db-extractor-config/blob/bc53baa05cd8ff7a2db4aeaad15614d1bdc2f1ea/tests/ConfigTest.php#L169-L174